et_replay/comm/backend/pytorch_dist_backend.py

```diff
@@ -1090,9 +1069,19 @@ def initialize_backend(
     def initialize_groups(self, backend="gloo"):
         groups = {}
         world_size = self.get_world_size()
-
+        global_rank = self.get_global_rank()
```

Comment:
Adding comment- this fixes issues/hang with multiple comm groups

Reply:
Hi, @briancoutinho
you mean, we should add this comment to source code, right?

Reply:
Done.